store/mockstore/mocktikv/rpc.go

```diff
@@ -723,9 +733,21 @@ func (c *RPCClient) checkArgs(ctx context.Context, addr string) (*rpcHandler, er
 	return handler, nil
 }
 
-// TiDBRPCServerCoprocessorHandler is the TiDB rpc server coprocessor handler.
+// GRPCClientFactory is the GRPC client factory.
+// Use global variable to avoid circle import.
 // TODO: remove this global variable.
-var TiDBRPCServerCoprocessorHandler func(context.Context, *coprocessor.Request) *coprocessor.Response
+var GRPCClientFactory func() Client
+
+func (c *RPCClient) redirectRequestToRPCServer(ctx context.Context, addr string, req *tikvrpc.Request, timeout time.Duration) (*tikvrpc.Response, error) {
+	if c.rpcCli == nil {
```

Comment:
I guess this function will be called concurrently?

Reply:
Great, done.